### PR TITLE
Translating TL;DR of Bureau Act describing the CEAP

### PR DESCRIPTION
**Pull request by hugonmelo**
Hello,

This is the reviewed .md for the portuguese version of the CEAP, along with the translated english version.

### CODE REVIEW

File: docs/CEAP.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+  * Líder de partido político, bloco parlamentar, da Minoria, ou do Governo na Câmara dos Deputados ou no Congresso Nacional.
+  * Vice-líder de Partido Político ou Bloco Parlamentar.
+  * Presidente ou Vice-Presidente de Comissão Permanente.
+  * Representantes de Partidos Políticos **com menos de um centésimo** da composição da Câmara dos Deputados.
+2. De R$ 902,02, o deputado que for:
+  * Vice-líder da Minoria.
+  * Vice-líder do Governo na Câmara dos Deputados ou no Congresso Nacional.
 * Se um mesmo deputado ocupa mais de um cargo simultaneamente, o valor **não se acumula**
-* Deputado que for representar o Brasil em missão no mercosul pode gastar mais 20% do valor da menor cota, por viagem
+* Deputado que for representar o Brasil em missão parlamentar do mercosul pode gastar mais 20% do valor da menor cota, por viagem
```
Comment (cuducos) [inline docs/CEAP.md]: A missão é **no** **M**ercosul (com letra maiúsculo), não *do* Mercosul, não?

File: docs/CEAP_en_US.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+Article 1 - It can use, above the calculated quota for it's State, an extra of:
+1. BRL 1353,04, the Congress Person who's:
```
Comment (cuducos) [inline docs/CEAP_en_US.md]: 1. In English locale the separator for decimals is a dot, not a comma. The comma is used as the thousands separator (e.g. 1,353.04)
2. It's **congressperson** (one word, no capital letter)
3. Not sure why but the contraction _who's_ (instead of _who is_) doesn't sound so familiar in that specific case (even if it might be grammatically correct)

File: docs/CEAP_en_US.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
+Article 1 - It can use, above the calculated quota for it's State, an extra of:
```
Comment (cuducos) [inline docs/CEAP_en_US.md]: Lot's of change needed here:

1. In the English language it's not common to invert the order of the sentence with the help of commas, so I wouldn't interpolate
2. In that case it is not **it's** (contraction of _it is_), it it **its** (possessive)
3. We do not refer to a given congressperson as **it**, but maybe as **one** or the impessoal **they**
4. State with capital letter usually refers to the concept of a (national) state, states as units of a federation does not need capital letter

That said I would rephrase it as _Above the calculated quota for their state they can use:_.

File: docs/CEAP_en_US.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+1. BRL 1353,04, the Congress Person who's:
+  * Leader of a political party, parliament block, Minority, or leader of the Government in the Chamber of Deputies or the National Congress.
```
Comment (cuducos) [inline docs/CEAP_en_US.md]: Does minority calls for capital letter?

File: docs/CEAP_en_US.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+  * Vice-leader of a political party or parliament block.
+  * President or vice-president of a Permanent Comission.
+  * Representative of political parties **with less members than a hundredth** of the total number of Congress People in the Chamber of Deputies.
```
Comment (cuducos) [inline docs/CEAP_en_US.md]: It's **congressperson** (one word, no capital letter)

File: docs/CEAP_en_US.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+2. BRL 902,02, the Congress Person who's:
```
Comment (cuducos) [inline docs/CEAP_en_US.md]: * 902.02 (dot, not comma)
* It's **congressperson** (one word, no capital letter)
* I suggest _who is_ instead of _who's_

File: docs/CEAP_en_US.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+  * Vice-leader of the Minority
+  * Vice-leader of the Government in the Chamber of Deputies or the National Congress.
+* If the same Congress Person occupies more than one of the above positions simultaneously, the extra quota **cannot be multiplied**.
```
Comment (cuducos) [inline docs/CEAP_en_US.md]: * It's **congressperson** (one word, no capital letter)
* No need for the comma (actually I know it's weird in pt, buy usual in Engli
* What about _the quotas are not cumulative (be multiplied sounds weird in this case)

File: docs/CEAP_en_US.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+* The Congress Person that represents Brazil in a Mercosul parliamentary mission can spend an extra 20% of the smallest quota, for each trip.
```
Comment (cuducos) [inline docs/CEAP_en_US.md]: * It's **congressperson** (one word, no capital letter)
* Moving the _For each trip_ to the beginning of the sentence makes it way clearer

File: docs/CEAP_en_US.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+
+Article 2 - The quota can be used to cover the following expenses:
+* Airline tickes
+* Telephony services
+* Postal services and products, but **not the purchase of postal stamps and shipping of postal cards**
```
Comment (cuducos) [inline docs/CEAP_en_US.md]: What about _**excluding** the purchase of postal stamps and shipping of postal cards_?

File: docs/CEAP_en_US.md [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+* Maintain an office to support the parliamentary activities, including:
```
Comment (cuducos) [inline docs/CEAP_en_US.md]: * **Maintenance** of an office to…
* No need for the comma before _including)

File: docs/CEAP.md (hunk 1 above)
Comment (hugonmelo) [inline docs/CEAP.md]: Done

File: docs/CEAP_en_US.md (hunk 3 above)
Comment (hugonmelo) [inline docs/CEAP_en_US.md]: Done.

File: docs/CEAP_en_US.md (hunk 2 above)
Comment (hugonmelo) [inline docs/CEAP_en_US.md]: Done

File: docs/CEAP_en_US.md (hunk 4 above)
Comment (hugonmelo) [inline docs/CEAP_en_US.md]: According to this source (http://www.camara.leg.br/Internet/Deputado/bancada.asp), the Minority is addressed as a political party and it uses capital M.

File: docs/CEAP_en_US.md (hunk 5 above)
Comment (hugonmelo) [inline docs/CEAP_en_US.md]: Done

File: docs/CEAP_en_US.md (hunk 6 above)
Comment (hugonmelo) [inline docs/CEAP_en_US.md]: Done

File: docs/CEAP_en_US.md (hunk 7 above)
Comment (hugonmelo) [inline docs/CEAP_en_US.md]: Done

File: docs/CEAP_en_US.md (hunk 8 above)
Comment (hugonmelo) [inline docs/CEAP_en_US.md]: Done

File: docs/CEAP_en_US.md (hunk 9 above)
Comment (hugonmelo) [inline docs/CEAP_en_US.md]: Done

File: docs/CEAP_en_US.md (hunk 10 above)
Comment (hugonmelo) [inline docs/CEAP_en_US.md]: Done

**Comment by cuducos** [comment]
@hugonmelo, once more thank you for all the time spent in translating the document. There many typos in the document and GitHub interface is not the ideal tool to correct them… so [I copied your text to Google Drive and added suggestions there: way easier to track](https://docs.google.com/document/d/1qvqhezE3lK-5Or9TK3NDwwd31hVHtjURaNeYA-A1SsA/edit).

Can you reflect those changes and corrections in you PR before I merge?

File: docs/CEAP.md (hunk 1 above)
Comment (guizero) [inline docs/CEAP.md]: Ha várias missões fora do Mercosul de parlamentares. Na missão da ONU em NYC, por exemplo, é muito comum parlamentares aparecerem por lá...

**Comment by guizero** [comment]
Fiz algumas sugestões em algumas partes também

**Comment by cuducos** [comment]
Valeu @guizero! Aceitei a maioria, mas algumas achei que mais prejudicavam o entendimento e recusei, tudo bem?

**Comment by hugonmelo** [comment]
Reviewed the changes listed in the google doc.

**Comment by cuducos** [comment]
Many thanks @hugonmelo! That's wonderful.

Before merging it we just need to link it somewhere. Basically we can link this English version in `CONTRIBUTING.md` and add a link to the Portuguese version in `CEAP_en.md`.

Actually as English is the official language of this repository I would even rename:

* `CEAP.md` => `CEAP_pt-BR.md`
* `CEAP_en.md` => `CEAP.md`

**Comment by hugonmelo** [comment]
@cuducos Should I upload another commit renaming the files?

**Comment by cuducos** [comment]
If you don't mind ; )

```console
$ git mv docs/CEAP.md docs/CEAP_pt_BR.md
$ git mv  docs/CEAP_en_US.md docs/CEAP.md
```

Add a fancy commit message, add a link to `CEAP_pt_BR.md` into `CEAP.mp`, commit with another fancy message, `git push` and we're good to merge ; )

**Comment by hugonmelo** [comment]
@cuducos I had to rebase my branch before committing, but it should be fine.

**Comment by cuducos** [comment]
Yay — great, many thanks ; )
